assets/js/frontend/give-checkout-global.js

```diff
@@ -66,30 +66,29 @@ jQuery(function ($) {
 	 * @since 1.2
 	 */
 	function format_cc_fields() {
-		give_form = $('form.give-form');
+		var $give_form = $('form.give-form');
 
 		//Loop through forms on page and set CC validation
-		give_form.each(function () {
-			var card_number = $(this).find('.card-number');
-			var card_cvc    = $(this).find('.card-cvc');
-			var card_expiry = $(this).find('.card-expiry');
+		$give_form.each(function (index, form) {
+			form            = $(form);
+			var card_number = form.find('.card-number'),
+				card_cvc    = form.find('.card-cvc'),
+				card_expiry = form.find('.card-expiry');
 
 			//Only validate if there is a card field
-			if (card_number.length === 0) {
```

Comment:
@DevinWalker main problem was with this condition. so if we have multiple forms on the same page and first donation form does not have any cc fields then instead of skipping the current form and process others, it was exiting the loop, so no formatting event fire for other forms.